### PR TITLE
Garret branch

### PR DESCRIPTION
**Pull request by GarretYang**
Added a contact us components

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -5,14 +5,15 @@
   "dependencies": {
     "axios": "^0.18.0",
     "bulma": "^0.7.4",
+    "bulma-modal-fx": "^1.1.1",
```
Comment (HuskyDanny) [inline client/package.json]: what is this package used for?

File: client/src/components/contactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Modal from 'react-responsive-modal';
+import "./contactUs.css";
+
+class ContactUs extends React.Component {
+
```
Comment (HuskyDanny) [inline client/src/components/contactUs.jsx]:
```suggestion
You could follow my signup logic here, for more details, please check my login.jsx
```
`<div style={{ padding: "4%" }}>
        <form onSubmit={this.handleSubmit}>
          <div className="field">
            <label className="label">Email</label>
            <div className="control has-icons-left has-icons-right">
              <input
                className="input is-primary"
                type="email"
                placeholder="Your Email"
                required
                onChange={e => this.handleChange(e, "email")}
                value={this.state.email}
              />

              <span className="icon is-small is-left">
                <i className="fas fa-envelope" />
              </span>
              {this.emailDuplicateError()}
            </div>
          </div>`

File: client/src/components/navbar.jsx [hunk 3]
```diff
@@ -38,10 +39,15 @@ const Navbar = props => {
             <div className="navbar-item has-dropdown is-hoverable">
               <a className="navbar-link">More</a>
 
-              <div className="navbar-dropdown">
+              <div className="navbar-dropdown buttons">
```
Comment (liuhantao9) [inline client/src/components/navbar.jsx]: Does this class name change all item in dropdown into buttons?

File: client/package.json (hunk 1 above)
Comment (GarretYang) [reply client/package.json]: It was for animations of the modal, but I did not use it at the end

File: client/src/components/contactUs.jsx (hunk 2 above)
Comment (GarretYang) [reply client/src/components/contactUs.jsx]: Ok, do you want me to add it and push the code again?

**Comment by GarretYang** [comment]
Added onChange and onSubmit handles to the "contactUs" component
Added input validation to the "contactUs" component

onSubmit does not have backend support yet (cant send emails)
started a new component - user profile

**Pull request merged**